renkuaqsannotation/plugin.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 - Viktor Gal
```

Comment:
we need to modify this license

Reply:
Which one shall I use instead?

Reply:
put yourself, remove the lines about " A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and.."

Reply:
no mention of UniGe and/or EPFL?

Reply:
put both in brackets after your name. It's not necessary for the license, and the requirements from the legal department do not specify this. we can adapt later common license.